main/telemetry.js

```diff
@@ -45,7 +45,7 @@ function setup () {
     // builds yet. As a result, Stations running on Apple Silicon were reporting
     // `x64` arch. To fix the problem, we detect `arm64` translation and report
     // a different architecture.
-    point.tag('arch', app.runningUnderARM64Translation ? 'arm64' : arch())
+    point.tag('arch', app?.runningUnderARM64Translation ? 'arm64' : arch())
```

Comment:
Under which circumstances can be `app` undefined?

Not a big deal; just trying to understand why we need this change.

Reply:
In the test suite, when we run `node` instead of `electron`

Reply:
Thanks for explaining!